Subject: [PATCH] Orion: DNS-323 C1 requires phylib

The code to support the DNS-323 rev C1 added a call to
phy_register_fixup_for_uid() and therefore phylib has to
be built in.

Signed-off-by: Martin Michlmayr <tbm@cyrius.com>

--- a/arch/arm/mach-orion5x/Kconfig	2010-07-15 22:54:05.000000000 +0000
+++ b/arch/arm/mach-orion5x/Kconfig	2010-07-15 22:54:16.000000000 +0000
@@ -26,6 +26,7 @@
 config MACH_DNS323
 	bool "D-Link DNS-323"
 	select I2C_BOARDINFO
+	select PHYLIB
 	help
 	  Say 'Y' here if you want your kernel to support the
 	  D-Link DNS-323 platform.
